lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -9,7 +9,7 @@
             'js/student_account/views/FormView',
             'text!../../../templates/financial-assistance/financial_assessment_form.underscore',
             'text!../../../templates/financial-assistance/financial_assessment_submitted.underscore',
-            'text!templates/student_account/form_field.underscore',
+            'text!lms/djangoapps/student_account/templates/student_account/form_field.underscore',
```

Comment:
@jcdyer Wow, that is confusing! That directory is a mixture of Mako templates which need to live in the templates directory, and Underscore files which need to be available to the browser. If the Underscore files don't use any Mako directives then I would move those to my proposed static directory, and leave the Mako templates where you had them.